test/functional/test_framework/test_node.py

```diff
@@ -99,15 +99,15 @@ def __init__(self, i, datadir, *, chain, rpchost, timewait, timeout_factor, bitc
         rpc_addr = self.init_rpc_url.split("http://")[-1].split('@')[-1]
         p2p_addr = p2p_url(self.index)
 
-        config_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "test_config.toml")
+        datadir_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "./.mintlayer")
```

Comment:
You don't have to use `./.mintlayer` in join. Just `.mintlayer` suffices. This is actually confusing.